.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,14 +1,17 @@
 blank_issues_enabled: false
 contact_links:
   - name: Bug Report
-    url: https://github.com/ethereum/solidity/issues/new?template=bug_report.md&projects=ethereum/solidity/43&labels=bug+%3Abug%3A
```

Comment:
As per @cameel 's [comment](https://github.com/ethereum/solidity/issues/13408#issuecomment-1219568910), the labels have been removed as they're not auto-applied anyway, likely due to users not having appropriate permissions.